jscomp/stdlib/buffer.ml

```diff
@@ -21,7 +21,10 @@ type t =
 
 let create n =
  let n = if n < 1 then 1 else n in
+#if BS then  
+#else 
```

Comment:
Note this may not be precise, we also need check cases like int overflow( check later)